php/PhpJsonApiSamples/nbproject/private/private.properties

```diff
@@ -3,6 +3,6 @@ copy.src.files=true
 copy.src.on.open=false
 copy.src.target=C:\\PhpCodeBackup
 debug.url=DO_NOT_OPEN_BROWSER
-index.file=Default.php
+index.file=Default.html

```

Comment:
I usually ignore IDE specific artifacts and don't commit it to github, but I suppose it's ok for a demo project. I just noticed this file for netbeans and I probably wouldn't use netbeans.

I wouldn't remove it now but just making a comment.